cmd/makenew/main.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+	"text/template"
+)
+
+const (
+	checkersPath = "../../lint/"
+)
+
+func main() {
+	name := flag.String("name", "", "name of a new checker")
+	flag.Parse()
+
+	if *name == "" {
+		log.Fatalf("empty checker name")
+	}
+
+	tmpl := template.Must(template.ParseFiles("new_checker.go.tmpl"))
+
+	buf := bytes.Buffer{}
+	err := tmpl.Execute(&buf, struct {
+		Name string
+	}{
+		Name: *name,
+	})
+	if err != nil {
+		log.Fatalf("render template: %v", err)
+	}
+
+	filename := checkersPath + *name + "_checker.go"
+
+	if err := ioutil.WriteFile(filename, buf.Bytes(), 0600); err != nil {
+		log.Fatalf("write output file: %v", err)
+	}
```

Comment:
You might want to call `format.Source` over the generated code.
Or run `gofmt` over the generated file.